hypershift-operator/controllers/manifests/ignitionserver/manifests.go

```diff
@@ -0,0 +1,121 @@
+package ignitionserver
+
+import (
+	routev1 "github.com/openshift/api/route/v1"
+	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/intstr"
+	k8sutilspointer "k8s.io/utils/pointer"
+)
+
+const (
+	resourceName = "ignition-server"
+)
+
+func Route(namespace string) *routev1.Route {
+	return &routev1.Route{
+		ObjectMeta: metav1.ObjectMeta{
+			Namespace: namespace,
+			Name:      resourceName,
+		},
+		Spec: routev1.RouteSpec{
```

Comment:
The manifest functions should only specify ObjectMeta — the Spec should be defined in the reconcile function.